README.md

```diff
@@ -74,13 +74,13 @@ they can take up a lot of space on your drive.*
 
 | [Squeak][squeak] | [Pharo][pharo]   | [GemStone][gemstone] | [Moose][moose]  | [GToolkit][gtoolkit] | 
 | ---------------- | ---------------- | -------------------- | --------------- | -------------------- |
-| `Squeak64-trunk` | `Pharo64-alpha`  | `GemStone64-3.5.x`   | `Moose64-trunk` | `GToolkit64-release` |
-| `Squeak64-6.0`   | `Pharo64-stable` | `GemStone64-3.4.x`   | `Moose64-10`    |                      |
-| `Squeak64-5.3`   | `Pharo64-11`     | `GemStone64-3.3.x`   | `Moose64-9.0`   |                      |
-| `Squeak64-5.2`   | `Pharo64-10`     | `GemStone64-3.2.x`   | `Moose64-8.0`   |                      |
-| `Squeak64-5.1`   | `Pharo64-9.0`    | `GemStone64-3.1.0.x` | `Moose64-7.0`   |                      |
-| `Squeak32-trunk` | `Pharo64-8.0`    | `Gemstone64-2.4.x`   | `Moose32-trunk` |                      |
```

Comment:
This is a breaking change. I'm assuming you're ok with that or should we do something about older GemStone versions?

Reply:
I'm not sure what can be done with older versions of GemStone ... they simply fail to run in 20.04 , if when someone hits that the only alternative is to remove the old version from spec ...

Reply:
smalltalkCI can be used on more than just GitHub Actions, someone may still run it on their own or some other Ubuntu 18.04 box, right?

Reply:
Oh, maybe we can't re-enable it because the client logic has been removed or does that still work?